src/deltacode/__init__.py

```diff
@@ -48,16 +49,38 @@ class DeltaCode(object):
     the form of File objects) contained in those scans.
     """
     def __init__(self, new_path, old_path, options):
-        self.new = Scan(new_path)
-        self.old = Scan(old_path)
+        self.codebase1 = None
+        self.codebase2 = None
+        self.new_files_fingerprint = dict() # map of { {new_file1:fingerprint},{new_file2:fingerprint},...} it will be needed when we need the fingerprints
+        self.old_files_fingerprint = dict() # map of { {old_file1:fingerprint},{old_file2:fingerprint},...}
+        self.new_files_original_path = dict() #this keeps a map of the path of file with respect to original path 
+
+        """ 
+            in self.new_files_fingerprint we keep the key as files original path and the value as its fingerprint value
+        """
+        self.old_files_original_path = dict()
```

Comment:
Same here, why do we need this dict again?

Reply:
The same reason as above is for keeping the old_files path and the above was for new_files.

Reply:
Lets just ignore/not do any alignment for this first implementation.